pkgs/development/ocaml-modules/patricia-tree/default.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildDunePackage,
+  fetchFromGitHub,
+  findlib,
+  mdx,
+  qcheck-core,
+  re,
+  camlp-streams,
+  csexp,
+  astring,
+  ocaml-version,
+  fmt,
+  logs,
+  ppx_inline_test,
+}:
+
+buildDunePackage rec {
+  pname = "patricia-tree";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "codex-semantics-library";
+    repo = "patricia-tree";
+    tag = "v${version}";
+    hash = "sha256-lpmU0KhsyIHxPBiw38ssA7XFEMsRvOT03MByoJG88Xs=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    mdx
+  ];
```

Comment:
```suggestion
  nativeCheckInputs = [
    mdx.mdx
  ];
```

Reply:
Sorry, there was a typo in my previous comment. The differences are twofold: `mdx` is a check-input (so not kept as a dependency in case you set `doCheck` to false); and only the binary is needed on the “native” platform.